website/dashboard/test_plots.js

```diff
@@ -0,0 +1,54 @@
+document.addEventListener('DOMContentLoaded', function() {
+    fetch('report-summary.json')
+        .then(response => response.json())
+        .then(data => {
+            function createPlot(container, parameter) {
+                var reference_values = data.map(d => d[parameter]);
+                var fit_values = data.map(d => d[parameter + '_fit']);
+                var errors = fit_values.map((d, i) => d - reference_values[i]);
+
+                var tolerance = reference_values.map((d, i) => data[i]['atol'][parameter] + data[i]['rtol'][parameter] * d);
+                var negative_tolerance = reference_values.map((d, i) => -(data[i]['atol'][parameter] + data[i]['rtol'][parameter] * d));
+
+                var scatter_trace = {
+                    x: reference_values,
+                    y: errors,
+                    mode: 'markers',
+                    type: 'scatter',
+                    name: parameter.toUpperCase() + ' Fit Errors'
+                };
+
+                var tolerance_trace = {
+                    x: reference_values,
```

Comment:
This should be data min to data max or just fixed numbers

Reply:
That's how it looks like after the changes:
<img width="389" alt="image" src="https://github.com/user-attachments/assets/e821e665-38d4-45e1-9e19-4a6d2d6f302a">